### DISCUSSION TITLE
Using PrimeOne-2.1.1 with Apollo theme

### DISCUSSION THREAD

**Discussion by akirichev** [primefaces/community#228] [Figma UI Kit] [answered]
Hello, I've purchased the Figma UI kit because my client is using the [Apollo theme ](https://www.figma.com/file/zQOW0XBXdCTqODzEOqwBtt/Preview-%7C-Apollo-2022?type=design&node-id=335-21768&mode=design&t=FL1jG29B9pCtWMOd-0). I am unable to find a way how to combine the components from the PrimeOne UI Kit with the Apollo since the Apollo is read-only.
How to get the over-styling that is specific for Apollo?

**Comment by onursenture** [comment, marked as answer]
Hello,style in Apollo already closely resembles PrimeOne, even if it's not an exact match. If you're talking about the Apollo Figma file when you mention 'read-only,' you could ask your client to download the Apollo Figma file from PrimeStore and send it to you. However, I must point out that you won't find UI component styles like Button / Input etc. in our Premium Templates' Figma files, just the designs for the Dashboard and other pages. Improving on this is something we'd like to work on in the future, although it's unfortunately not in our near-term roadmap at the moment.
